dashboard/src/pages/apps/new.tsx

```diff
@@ -0,0 +1,564 @@
+      </InputForm>
+    </>
+  )
+}
+
+export default () => {
+  const appsByRepo = () =>
+    loaded() &&
+    apps().applications.reduce((acc, app) => {
+      if (!acc[app.repositoryId]) acc[app.repositoryId] = []
+      acc[app.repositoryId].push(app)
+      return acc
+    }, {} as Record<string, Application[]>)
+
+  const [buildConfig, setBuildConfig] = createSignal(buildConfigItems[0].value)
+  const [websites, setWebsites] = createSignal<Website[]>([])
+  const [portPublications, setPortPublications] = createSignal<PortPublication[]>([])
+  const [searchParams] = useSearchParams()
+  const SelectRepository = (): JSX.Element => {
+    return (
+      <>
+        <ContentContainer>
+          <MainContentContainer>
+            <RepositoriesContainer>
+              {loaded() &&
+                repos()
+                  .repositories.filter((r) => r.id === searchParams.repositoryID)
+                  .map((r) => <RepositoryNameRow repo={r} apps={appsByRepo()[r.id] || []} onNewAppClick={() => {}} />)}
+            </RepositoriesContainer>
+
+            <InputFormContainer>
+              <InputForm>
+                <InputFormText>Application Name</InputFormText>
+                <InputBar placeholder='name' />
+              </InputForm>
+
+              <InputForm>
+                <InputFormText>RepositoryID</InputFormText>
+                <InputBar placeholder={searchParams.repositoryID} />
+              </InputForm>
+
+              <InputForm>
+                <InputFormText>Branch Name</InputFormText>
+                <InputBar placeholder='master' />
+              </InputForm>
+
+              <InputForm>
+                <InputFormTextBig>Build Config</InputFormTextBig>
+                <InputFormRadio>
+                  <InputForm>
+                    <Radio items={buildConfigItems} selected={buildConfig()} setSelected={setBuildConfig} />
+                  </InputForm>
+                  <Show when={buildConfig() === buildConfigItems[0].value}>
+                    <InputFormRuntimeConfig />
+                    <InputForm>
+                      <InputFormText>Context</InputFormText>
+                      <InputBar placeholder='context' />
+                    </InputForm>
+                  </Show>
+                  <Show when={buildConfig() === buildConfigItems[1].value}>
+                    <InputFormRuntimeConfig />
+                    <InputForm>
+                      <InputFormText>Base image</InputFormText>
+                      <InputBar placeholder='base_image' />
+                    </InputForm>
+                    <InputForm>
+                      <InputFormText>Build cmd</InputFormText>
+                      <InputBar placeholder='build_cmd' />
+                    </InputForm>
+                    <InputForm>
+                      <InputFormText>Build cmd shell</InputFormText>
+                      <InputFormCheckBox>
+                        <Checkbox>build_cmd_shell</Checkbox>
```

Comment:
例えばこの辺では、`build_cmd_shell` というunderscoreで繋がれている単語は、単に「protobufのフィールド名（or jsのフィールド名）がスペースを含んではいけない」という技術的な制約によってunderscoreで繋がれているだけなので、ユーザーにそれをそのまま出すのは変です

せめて上のラベルと合わせてスペースつなぎにするとか
```suggestion
                        <Checkbox>Build cmd shell</Checkbox>
```
もっと分かりやすく説明できるのであれば、そうするのが基本です
オプションの意味がわからなければコードを読むか、聞いてもらえると助かります
```suggestion
                        <Checkbox>Run build cmd with shell</Checkbox>
```